discovery-infra/day2.py

```diff
@@ -84,6 +87,8 @@ def day2_nodes_flow(client, cluster_name, cluster, image_path, num_worker_nodes,
     )
 
     if install_cluster_flag:
+        config_etc_hosts(api_vip_ip, api_vip_dnsname)
+        ocp_orig_ready_nodes = get_ocp_cluster_ready_nodes_num()
```

Comment:
lets have logs that says something before we start waiting

Reply:
fixed